src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -464,6 +466,17 @@ public void delete(DeleteRequest request, ActionListener<DeleteResponse> listene
             restHighLevelClient.deleteAsync(request, options, listener);
         }
 
+        /**
+         * Deletes a document from the index based on the query.
+         *
+         * @param request The delete by query request
+         * @param listener A listener to be notified with a result
+         *
+         */
+        public void deleteByQueryAsync(DeleteByQueryRequest request, ActionListener<BulkByScrollResponse> listener) {
```

Comment:
This is also different than all the others by adding `Async` to it.  Is this how it's called from extensions now?

Reply:
Since this has been merged. I'll take care of it in the next one.